swan/python/jupyter_server_config.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+c.NotebookNotary.db_file = os.environ.get("JUPYTER_PATH") + "/nbsignatures.db"
+c.NotebookNotary.secret_file = os.environ.get("JUPYTER_PATH") + "/notebook_secret"
+
+c.ServerApp.root_dir = os.environ.get("HOME")
+
+c.ServerApp.contents_manager_class = "swancontents.filemanager.SwanEosFileManager"
+
+root_data_dir = os.environ.get("ROOT_DATA_DIR")
+if root_data_dir is not None:
+  c.ServerApp.extra_static_paths = [root_data_dir + "/js"]
+
+# Convert the _xsrf cookie into a session cookie, to prevent it from having an expiration date of 30 days
```

Comment:
We should remove this and add it to a list of things to test.
From the top of my head, in the list we have:
- xsrf cookie setting
- ROOT without libxft

Reply:
Removed and added to the document.